kieserver/image.yaml

```diff
@@ -146,6 +146,9 @@ envs:
     - name: "KIE_SERVER_ROUTER_PORT"
       example: "9000"
       description: "KIE server router port (Used to set the org.kie.server.router system property)"
+    - name: "PROMETHEUS_SERVER_EXT_DISABLED"
```

Comment:
shouldn't we add it for others images as well?

Reply:
@spolti That property is only for the kieserver images. But the AB stuff comes from cct_module and is for all EAP-based images.